cmake/external/c-ares.cmake

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include(ExternalProject)
+if(TARGET c-ares)
```

Comment:
IWYU might suggest keeping the include. If you keep it, do so throughout.

Optional.

Reply:
You're right, sorry. I've put it back.

I had removed it because CMake's `include` mechanism is a simple textual replacement without any special support for preventing multiple inclusion. This wouldn't be so bad except that some CMake constructs cannot be redeclared, targets principal among them. An equivalent to C header guards is not common practice in CMake :-(.